cfme/tests/candu/__init__.py

```diff
@@ -25,3 +25,61 @@ def compare_data(table_data, graph_data, legends, tolerance=1):
                     else:
                         logger.warning("No {leg} data captured for DateTime: {dt}".format(
                             leg=leg, dt=row['Date/Time']))
+
+
+def compare_data_with_unit(table_data, graph_data, legends, tolerance=1):
+    """ Compare Utilization graph and table data and consider units
+    Args:
+        table_data : Data from Utilization table
+        graph_data : Data from Utilization graph
+        legends : Legends in graph; which will help for comparison
+        tolerance : Its error which we have to allow while comparison
+
+    Note: Mainly, when we check graph for some tag the unit in table reading missing. The unit
+        conversion totally depends on manual observation.
+
+    BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1367560
+    """
+    for row in table_data:
+        for key, data in graph_data.items():
+            if any([re.match(key, item) for item in row["Date/Time"].split()]):
+                for leg in legends:
+                    table_item = re.split(" |%", row[leg].replace(",", ""))
+                    tb_value, tb_unit = table_item if len(table_item) > 1 else (table_item[0], None)
+                    if table_item:
+                        graph_item = re.split(" |%", data[leg].replace(",", ""))
+                        gp_value, gp_unit = (
+                            graph_item if len(graph_item) > 1 else (graph_item[0], None)
+                        )
+
+                        if not tb_unit:
```

Comment:
I suggest you take a look at utilizing bitmath for this instead:

https://pypi.org/project/bitmath/1.0.2-3/

Reply:
@mshriver: I had experience with this lib; Its overall about `bits` conversions. I am not getting other usages of it in our `integration test` except this `workaround`. But, As its supporting `SI` and `NSI` both unit types. It's possible to play with `frequency` conversion also a little bit `str` parsing required.

I think it will be helpful for `storage` and other `utilization` scope; I am trying to play with `SI` unit of it with string parsing. If success, I will include this lib in requirements else we will move with a workaround.

Reply:
I tried with it but fails. I found this `bitmath` lib only supporting `bit` related operation not supporting other units like `MHz`. It will not provide a complete solution for this problem.
```
In [52]: x = bitmath.parse_string('1 GiB')
In [53]: x.MiB
Out[53]: MiB(1024.0)
In [54]: x = bitmath.parse_string('1 GB')
In [55]: x.MB
Out[55]: MB(1000.0)
In [56]: x = bitmath.parse_string('1 GHz')
---------------------------------------------------------------------------
ValueError 
```
Also, main issue will raise when there is no unit for table data. In this case we have again go with observation and conditions. 
Finally, I am on the decision that better go with the normal workaround.
@mshriver  what you think?